plugins/module_utils/common.py

```diff
@@ -663,6 +677,14 @@ def build_error_msg(kind, name, msg):
                 return result
 
         else:
+            if label_selectors:
+                filter_selector = LabelSelectorFilter(label_selectors)
+                if not filter_selector.isMatching(definition):
+                    result['changed'] = False
+                    del result['result']
```

Comment:
If we delete the result key, this is a breaking change. It should just be an empty dict.